### ISSUE TITLE
Could you add hvac_action attribute?

### ISSUE THREAD

**Issue by ddeconin-gh** [ollo69/ha-melcloud-custom#15]
Hi,
I noticed some other climate integrations supply a state attribute hvac_action.
Example: 
hvac_action: heating
Apparently this is taken into account by the thermostat card to make the termostat glow orange and the target temperature orange if supplied.
examples below. Without & with the hvac_action supplied

![image](https://github.com/ollo69/ha-melcloud-custom/assets/81985184/12987e2b-1c98-4386-8024-85bc540a5492)

**Comment by ollo69** [comment]
No, I cannot, because API is not returning this value. It should be based on the comparison between current temperature and target temperature, but this do not guarantee that we know the real action performed by the device.

**Comment by ddeconin-gh** [comment]
> No, I cannot, because API is not returning this value. It should be based on the comparison between current temperature and target temperature, but this do not guarantee that we know the real action performed by the device.

Isn’t is just as simple as setting it based on wether the climate is set to heat or cool? The example screenshot where this is done is actually coming from the same airco but instead of  through the MELCloud through a niko home control integration that itself integrates to MELCloud, so it essentially has (only) access to the same info…

**Comment by ollo69** [comment]
No, it isn't, otherwise it should be identical to `hvac_mode`. `hvac_action` is a dedicated property that should represent the current action. See https://developers.home-assistant.io/docs/core/entity/climate#hvac-action

**Comment by ollo69** [comment]
I just discovered that for Air-to-Water zone device this information is available, so I will add this with next release. I don't know if this is your case, unfortunately this information is not available for Air-to-Air devices
